src/shared/acpi-fpdt.c

```diff
@@ -61,6 +61,34 @@ struct acpi_fpdt_boot {
         uint64_t exit_services_exit;
 } _packed;
 
+/* /dev/mem is deprecated on many systems try using /sys/firmware/acpi/fpdt parsing instead */
+static int acpi_get_boot_usec_kernel_parsed(usec_t *ret_loader_start, usec_t *ret_loader_exit) {
+        usec_t nsecs;
+        int r;
+
```

Comment:
`assert(ret_loader_start)` and `assert(ret_loader_exit)`

Reply:
Ok, I will buy that, but should it be done in the caller?

Reply:
NM, its not an assert there

Reply:
I'm about to push a tweak which duplicates the caller's behavior.